apisix/plugins/limit-conn/init.lua

```diff
@@ -22,20 +22,17 @@ if ngx.config.subsystem == "stream" then
     shdict_name = shdict_name .. "-stream"
 end
 
-
```

Comment:
Why do you remove this blank,  two blanks are needed between different functions